ui/tests/acceptance/oidc-config/assignments-test.js

```diff
@@ -0,0 +1,198 @@
+import { module, test } from 'qunit';
+import { visit, currentURL, click, fillIn, findAll, currentRouteName } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import ENV from 'vault/config/environment';
+import authPage from 'vault/tests/pages/auth';
+import { OIDC_BASE_URL, SELECTORS } from 'vault/tests/helpers/oidc-config';
+import logout from 'vault/tests/pages/logout';
+import { create } from 'ember-cli-page-object';
+import fm from 'vault/tests/pages/components/flash-message';
+import {
+  clearRecord,
+  overrideCapabilities,
+  overrideMirageResponse,
+  ASSIGNMENT_LIST_RESPONSE,
+  ASSIGNMENT_DATA_RESPONSE,
+} from '../../helpers/oidc-config';
```

Comment:
I need to fix this in the oidc clients acceptance tests, too (because vscodes auto-import doesn't connect that `vault/tests` is the same as `../../`) but these imports can actually be combined and added to line 7.

Reply:
Yep! I caught that on Linda's PR but not my own. Thanks!